scripts/bioconductor/bioconductor_skeleton.py

```diff
@@ -179,7 +179,7 @@ def description(self):
         t = tarfile.open(self.cached_tarball)
         d = t.extractfile(os.path.join(self.package, 'DESCRIPTION')).read()
         self._contents = d
-        c = configparser.ConfigParser()
+        c = configparser.ConfigParser(strict=False)
```

Comment:
The DESCRIPTION of this package contains a duplicate key. Strict parsing would then fail to create the recipe.